### PR TITLE
docs: add readme

### LINKED ISSUE #9: Needs README

### PR DESCRIPTION
**Pull request by dignifiedquire**
Closes #9

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+sudo: false
+
+go:
+  - 1.7
+
+before_install:
+  - go get github.com/mattn/goveralls
+  - go get golang.org/x/tools/cmd/cover
+  - go get github.com/whyrusleeping/gx
+  - go get github.com/whyrusleeping/gx-go
```
Comment (lgierth) [inline .travis.yml]: Should be just `go get -d`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# go-ws-transport
+
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+[![GoDoc](https://godoc.org/github.com/ipfs/go-ws-transport?status.svg)](https://godoc.org/github.com/ipfs/go-ws-transport)
+[![Coverage Status](https://coveralls.io/repos/github/ipfs/go-ws-transport/badge.svg?branch=master)](https://coveralls.io/github/ipfs/go-ws-transport?branch=master)
+[![Build Status](https://travis-ci.org/ipfs/go-ws-transport.svg?branch=master)](https://travis-ci.org/ipfs/go-ws-transport)
```
Comment (lgierth) [inline README.md]: The badges all point to ipfs/go-ws-transport, should be libp2p

Comment (dignifiedquire) [inline README.md]: ups

File: .travis.yml (hunk 1 above)
Comment (dignifiedquire) [inline .travis.yml]: copied this from `go-libp2p-transport`, so should be changed there as well probably

File: README.md (hunk 2 above)
Comment (dignifiedquire) [inline README.md]: fixed

File: .travis.yml (hunk 1 above)
Comment (dignifiedquire) [inline .travis.yml]: fixed

Comment (lgierth) [inline .travis.yml]: Eeeh I'm sorry the way it was, was absolutely correct. This just installs the gx/test tooling, not actual code dependencies (which go get -d is for).

Could you change it back? Sorry :D

Comment (dignifiedquire) [inline .travis.yml]: 🤣 sure changed back :)

**Comment by lgierth** [comment]
Thanks :+1:

**Pull request merged**